actions/chats.js

```diff
@@ -54,6 +62,13 @@ export const addReaction = (data) => {
     };
 };
 
+export const setFoundUser = (data) => {
+    return {
+        type: types.SET_FOUND_USER,
+        payload: data
+    };
+};
```

Comment:
So I can keep track of the found user upon making a fetch GET request to `/user/:email`.